lib/config/presets/local/index.ts

```diff
@@ -24,6 +24,7 @@ const resolvers = {
   gitea,
   github,
   gitlab,
+  local: null,
```

Comment:
This means if someone runs locally against a repo with (valid) `local>` presets then it will fail